fpga/src/main/scala/vcu118/TestHarness.scala

```diff
@@ -84,8 +84,8 @@ class VCU118FPGATestHarness(override implicit val p: Parameters) extends VCU118S
     name = "chip_ddr",
     sourceId = IdRange(0, 1 << dp(ExtTLMem).get.master.idBits)
   )))))
-  ddrNode := ddrClient
-
+  ddrNode := TLWidthWidget(dp(XLen)/8) := ddrClient
```

Comment:
I think this might need to be `p(ExtTLMem).beatBytes`. Can you try it with this change?

Reply:
That's reasonable. I tried the change, and both RV 32&64 design worked on vcu118. :)

Reply:
Thanks!